src/core_plugins/kibana/public/management/sections/indices/create_index_pattern_wizard/index.js

```diff
@@ -1 +1,36 @@
-import './create_index_pattern_wizard';
+import { SavedObjectsClientProvider } from 'ui/saved_objects';
+import uiRoutes from 'ui/routes';
+import angularTemplate from './angular_template.html';
+import 'ui/index_patterns';
+import { documentationLinks } from 'ui/documentation_links';
+
+import { renderCreateIndexPatternWizard, destroyCreateIndexPatternWizard } from './render';
+
+uiRoutes.when('/management/kibana/index', {
+  template: angularTemplate,
+  controller: function ($scope, $injector) {
+    // Wait for the directives to execute
+    $scope.$$postDigest(() => {
+      const Notifier = $injector.get('Notifier');
+      const $routeParams = $injector.get('$routeParams');
+      const services = {
+        config: $injector.get('config'),
+        es: $injector.get('es'),
+        indexPatterns: $injector.get('indexPatterns'),
+        savedObjectsClient: $injector.get('Private')(SavedObjectsClientProvider),
+        kbnUrl: $injector.get('kbnUrl'),
+        notify: new Notifier(),
```

Comment:
Looks like this isn't in use? I think it was originally needed to surface a fatal error but yours truly removed that requirement. ;)

FYI, we can use the new `toastNotifications` service to show a toast when the index pattern is successfully created.

Reply:
Great point! Removed